apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -430,6 +430,7 @@ check_blocktime(_Config) ->
     {ok, TopBlock} = rpc(?NODE1, aec_chain, top_key_block, []),
     check_blocktime_(TopBlock).
 
+%% TODO: rework this test - currently assumes naive block timing
```

Comment:
It only assumes that the chain never produces two blocks with less than block-time distance between them. That is a rather reasonable assumption, I guess. But maybe you want this on average or something... Some checks is possibly needed.